doc/developing/testing.md

```diff
@@ -0,0 +1,70 @@
+# Testing Adaguc-server
+
+Adaguc contains two types of tests:
+
+- Tests specifically for adaguc-server
+- Tests specifically for the Python wrapper/EDR functionality
+
+During the tests, adaguc can use either the SQLite or PostgreSQL backend.
+
+## Running tests in Docker via SQLite
+
+The main [Dockerfile](~/Dockerfile) contains a _test stage_ which executes `runtests.sh`. This means that all tests will be executed during a regular docker build. Running the tests in Docker is recommended especially for users that don't run Linux/amd64.
+
+To run all tests, trigger a build:
+```bash
+docker build --progress plain -t adaguc-server .
+```
+
+The build will fail if any of the tests have failed.
+
+
+## Running tests outside Docker via SQLite
+
+Running the tests outside of Docker is possible by executing the `./runtests.sh` script.
+
+> **⚠️ Note**: some tests might fail due to platform architecture differences between amd64 and arm64.
+
+## Using postgres
```

Comment:
This is nice documentation, Good job! Do you think we can make this work with docker in github actions?

Reply:
I'll make an issue to see if this is possible.

Reply:
Created issue: https://github.com/KNMI/adaguc-server/issues/400